docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -14,6 +14,12 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hls-player                  | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhls-player.svg)](https://badge.fury.io/js/%40100mslive%2Fhls-player)           |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-08-09
+Released: `@100mslive/hms-video-store@0.10.11`, `@100mslive/react-sdk@0.8.11`, `@100mslive/hls-player@0.1.11`
+
+### Fixed:
+- Enable video when camera permission is denied
```

Comment:
this is not very clear. `selectIsLocalVideoEnabled` returning true when permissions denied